src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -81,19 +82,12 @@ SolverInterfaceImpl::SolverInterfaceImpl(
       _accessorProcessRank(accessorProcessRank),
       _accessorCommunicatorSize(accessorCommunicatorSize)
 {
-  PRECICE_CHECK(!_accessorName.empty(), "This participant's name is an empty string. When constructing a preCICE interface "
-                                        "you need to pass the name of the participant as first argument to the constructor.");
-  PRECICE_CHECK(_accessorProcessRank >= 0,
-                "The solver process index needs to be a non-negative number, not: "
-                    << _accessorProcessRank << ". Please check the value given when constructing a preCICE interface.");
-  PRECICE_CHECK(_accessorCommunicatorSize >= 1,
-                "The solver process size needs to be a positive number, not: "
-                    << _accessorCommunicatorSize << ". Please check the value given when constructing a preCICE interface.");
+  PRECICE_CHECK(!_accessorName.empty(), "Accessor has to be named!");
```

Comment:
These error messages seem to be worsening (reverted changes). Maybe merge from develop?

Reply:
Check also the rest of the diff.